vars/buildImage.groovy

```diff
@@ -0,0 +1,79 @@
+/* TODO:
+- Harcoded paths at jenkins/debian/... for Dockerfile and debos file
+*/
+
+def call(Closure context) {
+
+    def config = [:]
+    context.resolveStrategy = Closure.DELEGATE_FIRST
+    context.delegate = config
+    context()
+
+    def name = config.name
+    def kernel_arch = config.archList
+    def debian_arch =  ["arm": "armhf",
```

Comment:
I think we need armel and armhf.  kernelCI still has several ARMv5 boards that will need armel.